integration-tests/jail/compile/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  chain_compiler:
+    build: .
+    volumes:
+      - ../../..:/root/chain_src
+      - ../disk:/root/disk
+    image: chain_compiler
+    ports:
+       - "1022:22"
+    command: /root/disk/go_compile.sh
+    environment:
+      RUST_BACKTRACE: 1
+      RUST_LOG: debug
```

Comment:
what is this?

Reply:
why not just `nix-shell -p ci-scripts/drone.nix cargo build` ?

Reply:
i'll change

Reply:
just do `docker run --rm -v ../../..:/root/chain_src -v ../disk:/root/disk cryptocom/chain:latest /bin/bash /chain/<PATH TO go_compile.sh in chain repo>`

one line instead of tens of them -- and no need to have one more custom docker image, open ssh port and strange things like that.

it'd also make it simpler if there are less volumes and relative paths